cmd/botkube-agent/main.go

```diff
@@ -239,8 +239,10 @@ func run(ctx context.Context) (err error) {
 	//    For example, if in both communication groups there's a Slack configuration pointing to the same workspace,
 	//	  when user executes `kubectl` command, one Bot instance will execute the command and return response,
 	//	  and the second "Sorry, this channel is not authorized to execute kubectl command" error.
+	commGroupIdx := 1
```

Comment:
Maybe it would be good to document that, similar as we have here: https://github.com/kubeshop/botkube-cloud/blob/main/docs/global-infrastructure.md#segment-analytics

For me, this index is a bit misleading, so it would be good to have that documented.


```json
[
	{
		"type": "track",
		"messageId": "0",
		"anonymousId": "cluster-id",
		"event": "Integration enabled",
		"timestamp": "2009-11-17T20:34:58.651387237Z",
		"properties": {
			"index": 1,
			"platform": "slack",
			"type": "bot"
		}
	},
	{
		"type": "track",
		"messageId": "1",
		"anonymousId": "cluster-id",
		"event": "Integration enabled",
		"timestamp": "2009-11-17T20:34:58.651387237Z",
		"properties": {
			"index": 2,
			"platform": "discord",
			"type": "bot"
		}
	},
	{
		"type": "track",
		"messageId": "2",
		"anonymousId": "cluster-id",
		"event": "Integration enabled",
		"timestamp": "2009-11-17T20:34:58.651387237Z",
		"properties": {
			"index": 1,
			"platform": "teams",
			"type": "bot"
		}
	}
]
```

Here it looks like the Discord is somehow the second instance. While it's just 1 instance but in the second group, so maybe `index` -> `groundId`?